Client/src/Models/Authentication/IUserAccountModel.tsx

```diff
@@ -14,7 +14,14 @@ export interface IUserAccountModel {
   organizationName: string,
 }
 
+export interface IUserDetailsModel extends IUserAccountModel {
+  password?: string
+}
+
 export const toLocalUserAccountModel = (remoteAccount: IUserAccountRemoteModel): IUserAccountModel => {
+  if (remoteAccount == null) {
```

Comment:
When does this occur?

Reply:
its just a safe check, otherwise it crashes but the caller maybe able to deal with null result, else it crashes